woocommerce/woocommerce-ios
# Delete Unused Fulfillment Code

Part of #3034.

The PR https://github.com/woocommerce/woocommerce-ios/pull/3453 removes the Fulfillment screen and adds the Mark Order Complete button to the Order Details screen. 

This removes the code related to the old Fulfillment screen. 

## Testing

Please test that you can still fulfill an order using the new Mark Order Complete button. 

1. Navigate to a Processing order.
2. Tap on Mark Order Complete.
3. Confirm in WC Admin that the order was changed to Completed.

## Submitter Checklist
